src/store/index.js

```diff
@@ -0,0 +1,8 @@
+import { configureStore } from "@reduxjs/toolkit"
+
+export const store = configureStore({
+    reducer: {
+        //TODO
```

Comment:
I am wondering is this PR still a draft? If so, we can mark it as one in the PR page.